CMakeLists.txt

```diff
@@ -0,0 +1,101 @@
+cmake_minimum_required(VERSION 3.19)
```

Comment:
what feature of 3.19 are you using that is not supported in 3.0 or 3.12?

Reply:
Multi-configuration (although this is backward compatible) and many improvements to the find scripts. Unfortunately we cannot support large number of backwards version as the maintenance load is large.